### PR TITLE
Support RHEL 7.x

### PR DESCRIPTION
**Pull request by jojimt**
Changes to make sure install works on RHEL as well.

### CODE REVIEW

**Comment by gaurav-dalvi** [comment]
how are we handling (if at all) licensing of RHEL ? considering we are going to create RHEL VMs for testing.

**Comment by jojimt** [comment]
We are definitely not providing licenses :)

File: roles/docker/tasks/redhat_install_tasks.yml [hunk 1]
```diff
@@ -9,11 +9,8 @@
     validate_certs: "{{ validate_certs }}"
 
 - name: add docker CS-engine repos (redhat)
-  shell: yum-config-manager --add-repo https://packages.docker.com/{{ item }}/yum/repo/main/centos/7
+  shell: yum-config-manager --add-repo https://yum.dockerproject.org/repo/main/centos/7/
```
Comment (gaurav-dalvi) [inline roles/docker/tasks/redhat_install_tasks.yml:12]: I hope this does not break current centos 7 provisioning !

Comment (jojimt) [reply roles/docker/tasks/redhat_install_tasks.yml:12]: I have tested with centos and rhel.

**Pull request merged**